src/components/DelegationDashboard/DelegateCard.tsx

```diff
@@ -100,10 +130,23 @@ const DelegateCard = ({ delegate, className, trackNum, disabled }: Props) => {
 						</div>
 					)}
 					{!!delegate?.dataSource?.includes(EDelegateSource.PARITY) && (
-						<div className='flex items-center space-x-3'>
+						<div className='flex items-center space-x-2 sm:space-x-3'>
 							<div className='flex items-center space-x-[6px]'>
-								<ParityTechIcon />
-								<span className='text-xs font-normal text-bodyBlue dark:text-blue-dark-high'>Polkadot</span>
+								<Image
+									src={'/assets/icons/polkadot-logo.svg'}
```

Comment:
just use isMobile keyword to check display instead of hidden classaname and two times initialization of <Image